README.md

```diff
@@ -23,109 +23,76 @@
-- NEST is fast and memory efficient. It makes best use of your multi-core
-  computer and compute clusters with minimal user intervention.
-- NEST runs on a wide range of UNIX-like systems, from MacBooks to
-  supercomputers.
-- NEST has minimal dependencies. All it really needs is a C++ compiler.
-  Everything else is optional.
-- NEST developers are using agile continuous integration-based workflows in
-  order to maintain high code quality standards for correct and reproducible
-  simulations.
-- NEST has one of the largest and most experienced developer communities of all
-  neural simulators. NEST was first released in 1994 under the name SYNOD and
-  has been extended and improved ever since.
 
-## License
+## Documentation
+
+Please visit our [online documentation](https://nest-simulator.readthedocs.io/en/stable/) for details on installing and using NEST.
 
-NEST is open source software and is licensed under the [GNU General Public
-License v2](https://www.gnu.org/licenses/old-licenses/gpl-2.0.en.html) or
-later.
 
-## Installing NEST
+## Cite NEST
 
-Please see the online [NEST Installation Instructions](http://www.nest-simulator.org/installation)
-to find out how to install NEST.
+If you use NEST as part of your research, please cite the *version* of NEST you used.
+The full citation for each release can be found on [Zenodo](https://zenodo.org/search?q=title%3ANEST%20AND%20-description%3Agraphical%20AND%20simulator&l=list&p=1&s=10&sort=publication-desc)
 
-## Getting help
+For general citations, please use
 
-- You can run the `help` command in the NEST interpreter to find documentation
-  and learn more about available commands.
-- For queries regarding NEST usage, please use the [NEST users mailing
-  list](https://www.nest-initiative.org/mailinglist/).
-- Information on the Python bindings to NEST can be found in
-  `${prefix}/share/doc/nest/README.md`.
-- For those looking to extend NEST, developer documentation on [Contributing to
-  NEST](https://nest-simulator.readthedocs.io/en/latest/contribute/index.html) is available.
+`Gewaltig M-O & Diesmann M (2007) NEST (Neural Simulation Tool) Scholarpedia 2(4):1430.`
 
-## Citing NEST
+## Contact
 
-Please cite NEST if you use it in your work.
 
-- You can find all the information for [citing NEST here](https://nest-simulator.readthedocs.io/en/latest/citing-nest.html)
+If you need help or would like to discuss an idea or issue,
+you can find [ways to get in touch here](https://nest-simulator.readthedocs.io/en/stable/community.html).
 
 
+## Contribute
+
+NEST is built on an active community and we welcome contributions to our code and documentation.
+
+You can create a [GitHub issue](https://github.com/nest/nest-simulator/issues/new/choose), for bug reports, feature requests, documentation improvements, or other issues.
+
+For working with NEST code and documentation, [see our documentation](https://nest-simulator.readthedocs.io/en/stable/developer_space/index.html#contribute-to-nest)
```

Comment:
Also here we could add more links, especially to highlight the non-documentation channels, too.
```suggestion
NEST is built on an active community and we welcome everyone on [our mailinglist](https://nest-simulator.org/mailinglist) with comments and questions.

For documentation improvements, feature requests, bug reports or other issues you can create a [GitHub issue](https://github.com/nest/nest-simulator/issues/new/choose) which will be discussed in our fortnightly [meetings](https://nest-simulator.org/videoconference).

Active contributions to our code and documentation are highly regarded and you can find important steps for working with NEST code and documentation in our [contribute-to-nest documentation](https://nest-simulator.readthedocs.io/en/stable/developer_space/index.html#contribute-to-nest)
```